ui/raidboss/data/05-shb/eureka/delubrum_reginae_savage.txt

```diff
@@ -9,295 +9,295 @@ hideall "--sync--"
-2323.1 "Death Scythe" #sync /:Grim Reaper:5747:/
-2326.1 "Death Scythe" #sync /:Grim Reaper:5747:/
-2329.1 "Death Scythe" #sync /:Grim Reaper:5747:/
-2332.1 "Death Scythe" #sync /:Grim Reaper:5747:/
-2335.1 "Death Scythe" #sync /:Grim Reaper:5747:/
-2338.1 "Death Scythe" #sync /:Grim Reaper:5747:/
+2029.9 "Ruins Golem x2" sync / 03:........:Ruins Golem:/ 
+2032.1 "--sync--" sync / 14:Bicolor Golem:5558:/ window 20,10
+2037.1 "Metamorphose" sync / 1[56]:Bicolor Golem:5558:/
+2069.9 "Ruins Golem x2" sync / 03:........:Ruins Golem:/ 
+2071.3 "--sync--" sync / 14:Bicolor Golem:5558:/ window 20,10
+2076.3 "Metamorphose" sync / 1[56]:Bicolor Golem:5558:/
+2109.7 "Ruins Golem x2" sync / 03:........:Ruins Golem:/ 
+2140.1 "Ruins Golem x2" sync / 03:........:Ruins Golem:/ 
+2160.2 "Ruins Golem x2" sync / 03:........:Ruins Golem:/ 
+2170.3 "Ruins Golem x2" sync / 03:........:Ruins Golem:/ 
+2180.3 "Ruins Golem x2" sync / 03:........:Ruins Golem:/ 
+2189.9 "Ruins Golem x2" sync / 03:........:Ruins Golem:/ 
+2199.9 "Ruins Golem x2" sync / 03:........:Ruins Golem:/ 
+2209.9 "Ruins Golem x2" sync / 03:........:Ruins Golem:/ 
+2220.0 "Ruins Golem x2" sync / 03:........:Ruins Golem:/ 
+2230.0 "Ruins Golem x2" sync / 03:........:Ruins Golem:/ 
+2240.0 "Ruins Golem x2" sync / 03:........:Ruins Golem:/ 
+2250.1 "Ruins Golem x2" sync / 03:........:Ruins Golem:/ 
+2260.0 "Ruins Golem x2" sync / 03:........:Ruins Golem:/ 
+2270.0 "Ruins Golem x2" sync / 03:........:Ruins Golem:/ 
+2280.1 "Ruins Golem x2" sync / 03:........:Ruins Golem:/ 
+2290.1 "Ruins Golem x2" sync / 03:........:Ruins Golem:/ 
+2299.7 "Grim Reaper Enrage" sync / 03:........:Grim Reaper:/ 
+2300.2 "Ruins Golem x2" sync / 03:........:Ruins Golem:/ 
+2305.0 "Death Scythe" #sync / 1[56]:Grim Reaper:5747:/
+2308.0 "Death Scythe" #sync / 1[56]:Grim Reaper:5747:/
+2310.1 "Ruins Golem x2" sync / 03:........:Ruins Golem:/ 
+2311.1 "Death Scythe" #sync / 1[56]:Grim Reaper:5747:/
+2314.1 "Death Scythe" #sync / 1[56]:Grim Reaper:5747:/
+2317.1 "Death Scythe" sync / 1[56]:Grim Reaper:5747:/ window 5,10
+2320.1 "Death Scythe" #sync / 1[56]:Grim Reaper:5747:/
+2323.1 "Death Scythe" #sync / 1[56]:Grim Reaper:5747:/
+2326.1 "Death Scythe" #sync / 1[56]:Grim Reaper:5747:/
+2329.1 "Death Scythe" #sync / 1[56]:Grim Reaper:5747:/
+2332.1 "Death Scythe" #sync / 1[56]:Grim Reaper:5747:/
+2335.1 "Death Scythe" #sync / 1[56]:Grim Reaper:5747:/
+2338.1 "Death Scythe" #sync / 1[56]:Grim Reaper:5747:/
 
 
 ### Trinity Seeker
 # -p 5AD3:4011.4 5A98:4079.5 5A99:4171.7 5A97:4265.8
 # -ii 1961 1963 5B33 5AB8 5AB9 5ABA 5ABB 5ABC 5AD1 5B2A 5B2B 5AC1 5AC2 5AC3 5AC4 5AC5 5AC6 5ACB 5BBC 5ACD
-4000.0 "--sync--" sync / 00:0839:The Theater of One will be sealed off/ window 20000,0
-4000.0 "--Reset--" sync / 00:0839:.*is no longer sealed/ window 0,2000 jump 0
+4000.0 "--sync--" sync / 00:0839::The Theater of One will be sealed off/ window 20000,0
+4000.0 "--Reset--" sync / 00:0839:.*is no longer sealed:/ window 0,2000 jump 0
```

Comment:
```suggestion
4000.0 "--Reset--" sync / 00:0839:.*is no longer sealed/ window 0,2000 jump 0
```

Why did this (and some others) get a trailing colon, but others did not?

Reply:
It looks like the issue is that I (thought I) deliberately did not touch `Reset` lines whose timestamps weren't 0. However, this actually sent it through the `gameLog` fixer, and this in turn shifted things one field over, since `0839` was already assumed to be changed in a different process. Fortunately that should be a relatively simple fix.